src/server/config.js

```diff
@@ -0,0 +1,31 @@
+require('dotenv').config();
+const debug = require('./debug');
+
+process
```

Comment:
If I remember correctly, we decided to extract it to a separate module, don't we?